Sample Data/ASIM/Linux_Authpriv_UserManagement_IngestedLogs.csv

```diff
@@ -0,0 +1,50 @@
+﻿TenantId,SourceSystem,"TimeGenerated [UTC]",Computer,"EventTime [UTC]",Facility,HostName,SeverityLevel,SyslogMessage,ProcessID,HostIP,ProcessName,MG,CollectorHostName,Type,"_ResourceId"
```

Comment:
high % of empty values coming for some fields, is this expected?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/18efc091-458e-4699-9b75-936ec2d44db5)

Reply:
Yes this is expected